mage.go

```diff
@@ -0,0 +1,49 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+	"github.com/magefile/mage/sh"
+)
+
+const versionStringEnvVarName = "POCKET_VERSION_STRING"
```

Comment:
Why not use golang factoring here?

E.g. https://golangbyexample.com/multiple-constant-declarations-go/

Reply:
No particular reason. Taking this into account.

Reply:
My preference is to change it.